samples/cellular/lwm2m_client/sample_description.rst

```diff
@@ -397,9 +407,140 @@ CONFIG_SENSOR_MODULE_LIGHT - Configuration for light reading
+* :file:`overlay-avsystem.conf` - Uses `Coiote Device Management`_ server.
+* :file:`overlay-avsystem-bootstrap.conf` - Uses Coiote in bootstrap mode.
+
+NB-IoT:
+
+* :file:`overlay-nbiot.conf` - Enables the use of NB-IoT.
+
+LwM2M v1.1:
+
+* :file:`overlay-lwm2m-1.1.conf` - Enables LwM2M v1.1 protocol version.
+
+Firmware update:
+
+* :file:`overlay-adv-firmware.conf` - Enables experimental Advanced Firmware Update object.
+* :file:`overlay-fota_helper.conf` - Enables faster response for evaluating FOTA.
+
+Location assintance:
+
+* :file:`overlay-assist-agnss.conf` - Enables A-GNSS assistance.
+* :file:`overlay-assist-cell.conf` - Enables cell-based location assistance.
+* :file:`overlay-assist-pgps.conf` - Enables P-GPS assistance in the sample.
+
+Location service requirerequire `Coiote Device Management`_ server and LwM2M v1.1.
+
+DTLS optimications:
+
+* :file:`overlay-dtls-cid.conf` - Enables DTLS Connection Identifier.
+
+Power savings:
+
+* :file:`overlay-aggressive-psm.con` - Enables optimized PSM setup and proprietary PSM mode.
+* :file:`overlay-lowpower.conf` - Disables serial console to bring the power consumption down.
+
+LwM2M v1.1 conformance testing:
+
+* :file:`overlay-lwm2m-1.1-core-interop.conf` - Allows running of Core Specific Objects Test cases.
+* :file:`overlay-lwm2m-1.1-object-interop.conf` - Allows running of Additional Objects Test cases.
+
+
 Additional configuration
 ========================
 
+Default configuration
+---------------------
+
+LwM2M configuration:
+
+* Protocol version: 1.0
+* Binding mode: Queue
+* Device Management server: Leshan Demo server
+* Security: Enabled with PSK and DTLS session caching
+* Registration life time: 12 hours
+* Coap ACK initial timeout: 4 seconds
+* Enable LwM2M tickless mode power optimization
+* LwM2M shell
+
+Modem configurations:
+
+* Network Mode: LTE-M with GNSS.
+* PSM: Enabled TAU 60 minutes, RAT 30 seconds
+* Paging window: LTE 1.28 seconds and NB-IoT 2.56 seconds
+* eDRX: Enabled, with request of 10.24 seconds on LTE and 20.48 seconds on NB-Iot.
+* TAU pre-warning enabled, notification triggers registration update and TAU will be sent with the update which decreases power consumption.
+
+DTLS Connection Identifier
+--------------------------
+
+Add :file:`overlay-dtls-cid.conf` for activate DTLS Connection Identifier.
+Connection Identifier is recommend enable because it will reduce DTLS reconnection data traffic and also solve network NAT
+timeout problems from client to server side.
+
+.. note::
+   This requires modem firmware version 1.3.5 or newer.
+
+Modem proprietary PSM
+---------------------
+
+Add :file:`overlay-aggressive-psm.conf` to enable optimized PSM setup and proprietary PSM mode.
+Configure disables eDRX because it will request 10 seconds RAT and 12 hours TAU period.
+Proprietary PSM enables power saving when network not allows PSM. Modem enters PSM
+state after configured RAT period when connection is released.
```

Comment:
```suggestion
Add the :file:`overlay-aggressive-psm.conf` overlay file to enable optimized PSM setup and proprietary PSM mode.
This configuration disables eDRX, because it will request 10 seconds RAT and 12 hours TAU period.
Proprietary PSM enables power saving when network does not allow PSM.
The modem enters the PSM state after the configured RAT period when the connection is released.
```